addons/binding/org.openhab.binding.exec/ESH-INF/thing/thing-types.xml

```diff
@@ -29,7 +29,7 @@
             <parameter name="interval" type="integer"  required="false">
                 <label>Interval</label>
                 <description>Interval, in seconds, the command will be repeatedly executed</description>
-                <default>60</default>
+                <default>0</default>
```

Comment:
Hopefully, because I guess the current situation will continue to confuse users and wastes their time investigating whats happening on their system...